wire/msgmerkleblock_test.go

```diff
@@ -37,7 +38,7 @@ func TestMerkleBlock(t *testing.T) {
 
 	// Ensure max payload is expected value for latest protocol version.
 	// Num addresses (varInt) + max allowed addresses.
-	wantPayload := uint32(1000000)
+	wantPayload := uint32(4000000)
```

Comment:
It seems like this ended up in the wrong commit.  I believe it should be in the "implement witness encoding/decoding for transactions" commit since the tests would fail on this commit.